ltk/ltk.lisp

```diff
@@ -1154,23 +1146,19 @@ event to read and blocking is set to nil"
      finally (return result)))
   |#
 
-;; basic tk object
 (defclass tkobject ()
-  ((name :accessor name :initarg :name :initform nil)
-   )
+  ((name :accessor name :initarg :name :initform nil))
   (:documentation "Base class for every Tk object"))
 
-;; basic class for all widgets
-(defclass widget(tkobject)
+(defclass widget (tkobject)
   ((master :accessor master :initarg :master :initform nil) ;; parent widget or nil
-   (widget-path :initarg :path :initform nil :accessor %widget-path)         ;; pathname to refer to the widget
-   (init-command :accessor init-command :initform nil :initarg :init-command)
-   )
+   (widget-path :initarg :path :initform nil :accessor %widget-path) ;; pathname to refer to the widget
+   (init-command :accessor init-command :initform nil :initarg :init-command))
   (:documentation "Base class for all widget types"))
 
-;; creating of the tk widget after creating the clos object
```

Comment:
I removed the comment, as there is already a documentation string that says the same thing.